aspnetcore/signalr/version-differences.md

```diff
@@ -67,6 +71,14 @@ ASP.NET Core SignalR now supports [streaming data](xref:signalr/streaming) from
 
 The ability to pass arbitrary state between clients and the hub (often called HubState) has been removed, as well as support for progress messages. There is no counterpart of hub proxies at the moment.
 
+### GlobalHost
+
+ASP.NET Core has Dependency Injection built into the framework. The `GlobalHost` object that was commonly used to get a `HubContext` via `GlobalHost.ConnectionManager.GetHubContext<MyHub>();` no longer exists and instead services should use DI to access the [`HubContext`](xref:signalr/hubcontext).
```

Comment:
A suggestion to avoid past tense for ASP.NET 4.x SignalR:
```suggestion
ASP.NET Core has dependency injection (DI) built into the framework. Services can use DI to access the [HubContext](xref:signalr/hubcontext). The `GlobalHost` object that is used in ASP.NET SignalR to get a `HubContext` doesn't exist in ASP.NET Core SignalR.
```